### PR TITLE
Fixing Everest for everyone outside MS by using Vale

### PR DESCRIPTION
**Pull request by beurdouche**
@msprotz  Pointing to F* branch `beurdouche_everest2` for now because F* master killed miTLS

### CODE REVIEW

**Comment by beurdouche** [comment]
Logs (build and test passed)
https://raw.githubusercontent.com/project-everest/ci-logs/master/everest-build-668083fd-4132.all
https://raw.githubusercontent.com/project-everest/ci-logs/master/everest-test-668083fd-4132.all

File: everest [hunk 1]
```diff
@@ -413,7 +423,24 @@ OCAML
...
+      prompt_yes write_nuget_bashrc true
+    else
+      echo "Hint: brew install nuget (OSX)"
+      echo "Hint: most recent versions available at https://dist.nuget.org"
+      exit 1
+    fi
+  fi
+  echo "... nuget found in PATH"
+
+  # We bundle SCons in Vale, so Windows just needs a Python, while other
```
Comment (msprotz) [inline everest]: I don't think that comment is true anymore btw

File: everest [hunk 2]
```diff
@@ -509,34 +531,43 @@ do_reset ()
   done
 }
 
+run_nuget () {
+  # This function is called in a subshell in a test context (via possibly_log)
+  # which means that -e is not inherited. Explicit failures here.
+  cmd=$1
+  nuget restore vale/tools/Vale/src/packages.config -PackagesDirectory vale/tools/FsLexYacc $cmd || return 1
```
Comment (msprotz) [inline everest]: don't need the || 1 since this is the last command of the function

File: everest [hunk 3]
```diff
@@ -509,34 +531,43 @@ do_reset ()
...
   cmd=$1
   if is_windows; then
     VARS=$(cygpath -d "$VS_BIN\\..\\vcvarsall.bat")
-    DIR=$(cygpath -d "$(pwd)"/Spartan)
+    DIR=$(cygpath -d "$(pwd)"/vale)
     CMD=$(cygpath -u "$COMSPEC")
     $CMD /C "cd $DIR && $VARS amd64 && scons.bat $cmd" || return 1
   else
-    scons -C Spartan $cmd || return 1
+    scons -C vale $cmd || return 1
```
Comment (msprotz) [inline everest]: ibid... don't need the || return 1 when this is the last command of the function

**Pull request merged**